linker.go

```diff
@@ -80,6 +81,24 @@ func findReferences(progs map[string]*ProgramSpec) error {
 	return nil
 }
 
+// collectCoreRelos returns a list of CO-RE relocations of the layout's progs
+// in order.
+func collectCoreRelos(layout []reference) btf.CoreRelos {
```

Comment:
Nit: we should decide on CORE or Core. I think CORE is more appropriate since it's an acronym.

Reply:
Yup, but then it needs changing across the board. Better now than never!